superset/daos/annotation.py

```diff
@@ -67,21 +51,6 @@ def validate_update_uniqueness(
 class AnnotationLayerDAO(BaseDAO):
     model_cls = AnnotationLayer
 
-    @staticmethod
-    def bulk_delete(
```

Comment:
Copypasta except the `BaseDAO` is defined as a `classmethod` as opposed to a `staticmethod`.

Reply:
Nice cleanup!